server/testdriver/core/src/main/java/org/infinispan/server/test/core/ContainerInfinispanServerDriver.java

```diff
@@ -82,7 +82,7 @@ public class ContainerInfinispanServerDriver extends AbstractInfinispanServerDri
    private static final Long IMAGE_MEMORY = Long.getLong(TestSystemPropertyNames.INFINISPAN_TEST_SERVER_CONTAINER_MEMORY, null);
    private static final Long IMAGE_MEMORY_SWAP = Long.getLong(TestSystemPropertyNames.INFINISPAN_TEST_SERVER_CONTAINER_MEMORY_SWAP, null);
    public static final String INFINISPAN_SERVER_HOME = "/opt/infinispan";
-   public static final String JDK_BASE_IMAGE_NAME = "registry.access.redhat.com/ubi8/openjdk-17-runtime";
+   public static final String JDK_BASE_IMAGE_NAME = "registry.access.redhat.com/ubi9/openjdk-21-runtime";
```

Comment:
should we enable virtual threads? `-Dorg.infinispan.threads.virtual=true`

Reply:
If we're going to enable them by default for our images, in the Operator etc, I would say yes. The testsuite should reflect our most commonly used configuration.

Reply:
@pruivo I think 15 should definitely enable vthreads by default (unlike 14).